bpf/kprobe_pwru.c

```diff
@@ -38,7 +38,8 @@ struct skb_meta {
 	u32 len;
 	u32 mtu;
 	u16 protocol;
-	u16 pad;
+	u8 pkt_type;
```

Comment:
AFAIK, the packet type should be available in `--output-skb` (it should print it if it's non-zero). Have you checked it?